.gitignore

```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# AppSetting environment files
```

Comment:
Did you mean to commit this?

Reply:
haven't changed this file knowingly, just checked the commit details and James King changed it. He did some style work for the button and added it to the repo so it looks to have come from that